events/wc_society_events.txt

```diff
@@ -173,3 +173,304 @@ character_event = {
 		set_character_flag = ask_for_ancient_mark_cooldown_flag
 	}
 }
+
+#Write down whispers
+character_event = {
+	id = WCCSO.10200
+	desc = EVTDESC_WCCSO.10200
+	picture = GFX_evt_madness
+    border = GFX_event_normal_frame_religion
+
+    is_triggered_only = yes
+
+    trigger = { can_use_society_power_write_whispers_trigger = yes }
+
+	fail_trigger_effect = { character_event = { id = WCCSO.10204 } }
+
+    option = {
+    	name = EVTOPTA_WCCSO.10200
+    	add_character_modifier = {
+    		modifier = writing_whispers
+			years = 2
+    	}
+		character_event = { id = WCCSO.10201 days = 90 random = 60 }
+	}
+}
+
+#Write down whispers - what to focus on
+character_event = {
+	id = WCCSO.10201
+	desc = EVTDESC_WCCSO.10201
+	picture = GFX_evt_madness
+    border = GFX_event_normal_frame_religion
+
+    is_triggered_only = yes
+
+    trigger = { can_use_society_power_write_whispers_trigger = yes }
+
+	fail_trigger_effect = { character_event = { id = WCCSO.10204 } }
+
+    option = { #martial
+    	name = EVTOPTA_WCCSO.10201
+    	trigger = {
+    		NOT = {
+    			has_artifact = whispers_martial_1
+				has_artifact = whispers_martial_2
+				has_artifact = whispers_martial_3
+				has_artifact = whispers_martial_4
+    		}
+    	}
+    	hidden_effect = {
+			set_character_flag = writing_whispers_martial
+    		character_event = { id = WCCSO.10202 days = 90 random = 60 }
```

Comment:
You can move it to `after` block.
https://ck2.paradoxwikis.com/Event_modding#After

Reply:
done